configuration.html

```diff
@@ -249,7 +251,7 @@
       <li class="pytorch-breadcrumbs-aside">
         
             
-            <a href="_sources/configuration.rst.txt" rel="nofollow"><img src="_static/images/view-page-source-icon.svg"></a>
+            <a href="_sources/configuration.md.txt" rel="nofollow"><img src="_static/images/view-page-source-icon.svg"></a>
```

Comment:
@jlin27 Is this the right file name "configuration.md.txt" ?

Reply:
Yep that's correct. It's because for the first website, I'd converted all the md files to rst files, but for this build, I was able to directly build from the md files.